mkdocs.yml

```diff
@@ -169,6 +164,13 @@ nav:
           - ReCaptcha: docs/reference/modules/ReCaptcha/README.md
           - Resources: docs/reference/modules/Resources/README.md
           - Rules: docs/reference/modules/Rules/README.md
+          - Search, Indexing, Querying:
```

Comment:
This is the same as the section on the page: https://docs.orchardcore.net/en/latest/docs/reference/#search-indexing-querying And that looks good so, since while these are tightly related topics, it's not just search (like SQL queries have nothing to do with search, but queries in general do relate to Lucene and Elasticsearch, which are about indexing, and usually, but not necessarily search...)